### PR TITLE
Feat/object detection webcam

### PR DESCRIPTION
**Pull request by leecr1215**
- users can try out the object detection model with their webcam
image

[Future improvements]
- object detection speed
- add socket

### CODE REVIEW

File: GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.html
Comment (ramaking) [inline GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.html:1]: please remove unnecessary comment 😉

**Review by ramaking — DISMISSED** [review]

**Comment by chickennight** [comment]
There are a lot of changes in package-lock.json, are they appropriate?

File: GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
+<script type="text/javascript">
+    RED.nodes.registerType("webcam-object-detection", {
+      category: "GOOD I DEER",
+      color: "#1AE5BE",
+      defaults: {
+        name: { value: "" },
+        model: { value: "YoloV8n"},
```
Comment (Gitgloo) [inline GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.html]: I think value must be "yolov8n", not "Yolov8n"

File: GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.html
Comment (leecr1215) [reply GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.html:1]: oh i got it

File: GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        threshold: {value: 0.5, validate: function(v){
+          let threshold = Number($("#node-input-threshold").val());
```
Comment (Gitgloo) [inline GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.html]: I think threshold must be 'v'

**Review by chickennight — DISMISSED** [review]

File: GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        function HTML() {
+            console.log("HTML()의 config: ", config);
+            return require('./webcam-html.js').code(config)
+        }
+        
+          
+        console.log('webcamYoloNode의 config')
+        console.log(config);
+        
+        // on input event
```
Comment (Gitgloo) [inline GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.js]: please remove comment

File: GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        node.on('input', (msg, send, done) => {
+            console.log('==============msg============');
+            console.log(msg);
+        
+            // return HTML document to client
```
Comment (Gitgloo) [inline GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.js]: please remove comment

File: GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+            msg.payload = HTML()
+            send = send || function() { this.send.apply(this, arguments )}
+            send(msg)
+        })
+
+        // flow closed
```
Comment (Gitgloo) [inline GOOD-I-DEER/nodes/object-detection-node/webcam-object-detection.js]: please remove comment

**Comment by leecr1215** [comment]
> There are a lot of changes in package-lock.json, are they appropriate?

Let me check it out. thank you

**Review by Gitgloo — APPROVED** [review]

**Review by ramaking — APPROVED** [review]

**Pull request merged**